docs/manual-EN/server-administration/storage-balance.md

```diff
@@ -0,0 +1,219 @@
+# Storage Balance Usage
+Nebula's services are composed of  three parts: graphd, storaged and metad. The **balance** in this document targets the storaged. Currently, storage is scaled by `balance` prodecure. There are two kinds of balance command, one is to move data, the command is `balance data`; the other one only changes the distribution of leader partition to balance load without moving data, the command is `balance leader`.
```

Comment:
targets-> forcus on

storaged can be scaled horizontally by the `balance` command.

one is to move data, which is `balance data`